src/Metadata/Property/Factory/SerializerPropertyMetadataFactory.php

```diff
@@ -155,6 +155,12 @@ private function getEffectiveSerializerGroups(array $options, string $resourceCl
         if (isset($options['serializer_groups'])) {
             $groups = (array) $options['serializer_groups'];
 
+            if (isset($options['deserializer_groups'])) {
+                $deserializerGroups = (array) $options['deserializer_groups'];
```

Comment:
`serialization_groups` and `deserialization_groups` or `normalization_groups` and `denormalization_context` (as `(de)normalization_context` that will already use will probably be more consistent; but it's just some nitpicking, I don't think deprecating the existing option is worth it.